app/models/score.rb

```diff
@@ -1,3 +1,4 @@
 class Score < ApplicationRecord
   belongs_to :composition
+  validates :score, :playlist_id, presence: true
```

Comment:
Sorry, I don't get why you have playlist_id there. you don' t have to validate presence for association.